tests/rbac_test.go

```diff
@@ -38,37 +38,37 @@ var _ = Describe("RBAC", func() {
 
 	AfterEach(func() {
 		By(fmt.Sprintf("Login with the user %s", tests.UsernameAdminUser))
-		_, err := ktests.RunCommand("oc", "login", "-u", tests.UsernameAdminUser, "-p", "123456")
+		_, _, err := ktests.RunCommand("oc", "login", "-u", tests.UsernameAdminUser, "-p", "123456")
 		Expect(err).ToNot(HaveOccurred(), "Should login as %s user", tests.UsernameAdminUser)
 
 		tests.DeleteUser(tests.UsernameTestUser)
 
 		By(fmt.Sprintf("Deleting the identity associated with %s user", tests.UsernameTestUser))
-		_, err = ktests.RunCommand("oc", "delete", "identity", "allow_all_auth:"+tests.UsernameTestUser)
+		_, _, err = ktests.RunCommand("oc", "delete", "identity", "allow_all_auth:"+tests.UsernameTestUser)
 		Expect(err).ToNot(HaveOccurred(), "Should delete identity of the %s user", tests.UsernameTestUser)
 	})
 
 	Describe("RBAC with default permission", func() {
 		It("should allow to access subresource endpoint.", func() {
 			By(fmt.Sprintf("Login with the user %s", tests.UsernameTestUser))
-			_, err := ktests.RunCommand("oc", "login", "-u", tests.UsernameTestUser, "-p", "123456")
+			_, _, err := ktests.RunCommand("oc", "login", "-u", tests.UsernameTestUser, "-p", "123456")
 			Expect(err).ToNot(HaveOccurred(), "Should login as %s user", tests.UsernameTestUser)
 
 			By("Creating a project/namespace")
-			_, err = ktests.RunCommand("oc", "new-project", NamespaceTestSystem)
+			_, _, err = ktests.RunCommand("oc", "new-project", NamespaceTestSystem)
 			Expect(err).ToNot(HaveOccurred(), "Should create %s project/namespace", NamespaceTestSystem)
 
 			By("Creating a VM from manifest")
-			_, err = ktests.RunCommand("oc", "create", "-f", vm.Manifest, "-n", NamespaceTestSystem)
+			_, _, err = ktests.RunCommand("oc", "create", "-f", vm.Manifest, "-n", NamespaceTestSystem)
```

Comment:
It is hard to say what happened there, without formatting stderr (second return value) into error message :-/

Reply:
I see that stderr is being logged here https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L1968 ... so I am curious why we don't see these log messages.

Reply:
I saw the following message:

```
{"command":"/usr/bin/oc create -f tests/manifests/vm-cirros.yaml -n kubevirt-test-system -n kubevirt-test-default","component":"","level":"error","msg":"command failed: cannot run command","output":"","pos":"utils.go:2004","reason":"exit status 1","stderr":"Error from server (Forbidden): error when creating \"tests/manifests/vm-cirros.yaml\": virtualmachines.kubevirt.io is forbidden: User \"kubevirt-test-user\" cannot create virtualmachines.kubevirt.io in the namespace \"kubevirt-test-default\": User \"kubevirt-test-user\" cannot create virtualmachines.kubevirt.io in project \"kubevirt-test-default\"\n","timestamp":"2018-10-29T13:08:43.489885Z"}
```

Reply:
@gbenhaim @lukas-bednar  The problem is [RunCommand function](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L1946) has changed and It allows to run all command with kubevirt-test-default namespace. As my test cases need to run with the different namespace and some test case are need to run with kubevirt-test-default namespace. 
Now the test is trying to create resources with both the namespace which is not allowed for my test and failure has occurred.

Reply:
@gbenhaim @lukas-bednar This will solve the issue https://github.com/kubevirt/kubevirt-ansible/pull/441